mod/common.php

```diff
@@ -42,7 +41,7 @@ function common_content(App $a)
 
 		if (DBA::isResult($contact)) {
 			$a->page['aside'] = "";
-			Profile::load($a, "", 0, Contact::getDetailsByURL($contact["url"]));
+			Model\Profile::load($a, "", 0,Model\Contact::getDetailsByURL($contact["url"]));
```

Comment:
Code standards: Please keep the space after commas.

Reply:
Sorry for all the Code standards mistakes. Was an find & replace error.

Reply:
No problem, take your time!